modules/dataquery/static/schema.yml

```diff
@@ -0,0 +1,353 @@
+openapi: 3.0.1
+info:
+  title: LORIS Data Query Tool API endpoints
+  description: |-
+    This is the API for the parts of the Loris data query module which are intended to be used as an API.
+  contact:
+    name: LORIS Development Team
+    url: 'https://github.com/aces/Loris'
+  license:
+    name: 'GNU Public License, Version 3'
+    url: 'https://opensource.org/licenses/GPL-3.0'
+  version: "3.0"
+servers:
+  - url: /dataquery/
+security:
+  - ApiKeyAuth: []
+paths:
+  /queries:
+    get:
+      summary: Get a list of a recent, shared, and study (top) queries to display.
+      responses:
+        '200':
+          description: Successfully operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AllQueries'
+        '403':
+          description: Permission denied
+    post:
+      description: Create a new query and return the QueryID. If the same query (fields and criteria) already exists, the same QueryID will be returned instead of a new one being created.
+      requestBody:
+        description: A Query object (without the QueryID)
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Query'
+      responses:
+        '200':
+          description: Query successfully created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  QueryID:
+                    type: integer
+        '400':
+          description: An invalid body was supplied
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+  /queries/{QueryID}:
+    parameters:
+      - name: QueryID
+        in: path
+        description: the QueryID returned by posting to /queries
+        required: true
+        style: simple
+        schema:
+          type: integer
+    get:
+      responses:
+        '200':
+          description: The Query was successfully retrieved
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Query'
+        '403':
+          description: Permission denied
+    patch:
```

Comment:
Can I send: 
share=true&unshared=true
?

Reply:
previously, yes, but you would get a bad request response. After latest update: no.

Reply:
why not use share=true|false and not use unshare?

Reply:
that is what's done in the latest update